PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+
+# What you've done
+...
+
+# Why have you done it
+...
+
+# Screenshot (if visual)
+...
+
+# Any testing carried out
+...
```

Comment:
Here is a sample 

Issue : https://github.com/gayanhewa/sailsjs-cacheman/issues/7